src/ussdapp.js

```diff
@@ -367,6 +370,193 @@ go.app = function() {
+                question: $("The system is looking up services near you. This usually takes less than a minute."),
+                choices: [
+                    new Choice('state_locate_get_results', $("View services"))
+                ],
+                next: function(choice) {
+                    return choice.value;
+                }
+            });
+        });
+
+        // state_locate_stall_again
+        self.add('state_locate_stall_again', function(name) {
+            return new ChoiceState(name, {
+                question: $("The system was still busy finding your services. Please try again now or choose Exit and dial back later."),
+                choices: [
+                    new Choice('state_locate_get_results', $("View services")),
+                    new Choice('state_locate_exit', $("Exit"))
+                ],
+                next: function(choice) {
+                    return choice.value;
+                }
+            });
+        });
+
+        // state_locate_exit
+        self.add('state_locate_exit', function(name) {
+            return new EndState(name, {
+                text: $('Please dial back in a few minutes to see your services results'),
+                next: 'state_locate_stall_again'
+            });
+        });
+
+
+        // state_locate_get_results
+        self.add('state_locate_get_results', function(name) {
+            return go.utils.get_poi_results(self.im, self.contact)
+            .then(function(poi_results) {
+                if (poi_results.length === 0) {
+                    // stall again if results are not available
+                    // TODO handle search complete but no results found
+                    return self.states.create('state_locate_stall_again');
+                } else {
+                    var opts = { poi_results: poi_results };
+                    return self.states.create('state_locate_show_results', opts);
+                }
+            });
+        });
+
+        // state_locate_show_results
+        self.add('state_locate_show_results', function(name, opts) {
+            var choices = [];
+            opts.poi_results.forEach(function(poi_result) {
+                choices.push(new Choice('state_locate_details', poi_result));
+            });
+
+            return new ChoiceState(name, {
+                question: $('Select a service for more info'),
+                choices: choices,
+                next: function(choice) {
+                    // TODO create state that shows the selected service's details after Api has
+                    // been implemented
+                    return choice.value;
+                }
+            });
+        });
+
+
+    // REFUGEE MENU STATES
+
+        // 024
+        self.add('state_024', function(name) {
+            return new ChoiceState(name, {
+                question: $('Select an option'),
+                choices: [
+                    new Choice('state_locate_me', $("Find nearest SService"))
```

Comment:
The content is not available yet so it'll have to change later in any case, but that's short for Support Service. It's just placeholder text for now.